ProjectHeads/App.Head.props

```diff
@@ -40,7 +40,18 @@
       </ItemGroup>
     </When>
   </Choose>
-
+  <Choose>
+    <When Condition="'$(ToolkitSettingsControlsSourceProject)' == ''">
+      <ItemGroup>
+        <PackageReference Include="CommunityToolkit.$(DependencyVariant).Controls.SettingsControls" Version="8.0.0-beta.1"/>
```

Comment:
There's two possible problems here:
- Version mismatch 8.0.230801-preview
- Transitive dependencies for the Settings Controls